week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/model/Hero.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface Hero {
+    title: string;
+    subTitle: string;
+}
```

Comment:
to jest "prop-type" dla `hero.component` i może mieszkać razem z `hero.component`

Reply:
Przeniosłem do hero.component i usunąłem ten plik.